test/algorithms/set_operations/difference/difference_linear_linear.cpp

```diff
@@ -270,12 +270,6 @@ BOOST_AUTO_TEST_CASE( test_difference_linestring_linestring )
          from_wkt<ML>("MULTILINESTRING((0 0,0.5 0))"),
          "lldf16");
 
-    tester::apply
```

Comment:
I understand that all the reverse tests `-r` are removed as duplicates and the rest are disactivated for now and fixed in the future. Are the `-r` tests failing now too? Are they going to be fixed in the future as well?

Reply:
- the `-r` cases are indeed removed as duplicate
- some of the `-r` cases are now failing, and only they are disabled
- yes, this was the first step to see what's going wrong there and I will continue next Wednesday
- so this PR is a first step - it was the last step in letting all tests pass, without rescaling, in set_operations and buffer

Reply:
I don't understand what is the rationale behind removing them? These are not duplicates since the inputs are different. The fact that some of them fails now is a good reason why they should stay. It's because they might fail in the future after changing something else. This is what tests are for, not to pass but to fail if something is wrong.

Reply:
@vissarion Ok thanks. So they are not removed but instead various `settings` like `test_normal_reverse` are used inside.

Reply:
Sure, conform description

The reverse cases where already tested inside. So they were always duplicate. But now they are configurable.